AI/StupidAI/StupidAI.cpp

```diff
@@ -49,7 +49,8 @@ struct EnemyInfo
 	const CStack * s;
 	int adi, adr;
 	std::vector<BattleHex> attackFrom; //for melee fight
-	EnemyInfo(const CStack * _s) : s(_s), adi(0), adr(0)
+	EnemyInfo(const CStack * _s)
+		: s(_s), adi(0), adr(0)

```

Comment:
Yes. We can also don't touch it at all.

@IvanSavenko I'm really against any alignment because it's become complete mess as soon as you use different IDE settings from author of the code. And even if formatter can do some alignment it's will be mess for one developer or another.